appinventor/components/src/com/google/appinventor/components/runtime/Ball.java

```diff
@@ -237,4 +239,15 @@ public double Y() {
   public void MoveTo(double x, double y) {
     super.MoveTo(x, y);
   }
+
+  /**
+   * Sets the `x` and `y` coordinates of the `Ball`. If {@link #OriginAtCenter(boolean)} is true,
+   * the center of the `Ball` will be placed here. Otherwise, the top left edge of the `Ball` will
+   * be placed at the specified coordinates.
+   * @param coordinates a list of length 2 where the first item is the x-coordinate and the
+   * second item is the y-coordinate.
+   */
+  public void MoveToPoint(YailList coordinates) {
```

Comment:
You don't need to override the super here. That's just a trick we use when we want to expose functions on some components but not others.